README.md

```diff
@@ -13,32 +13,43 @@ For a longer description and available features and options, see the
 
 # Installation
 
-Installing mkosi is easy, as it has no runtime Python dependencies (you will
-need all the tools to format filesystems and bootstrap the distribution
-appropriate for your image, though).
+You can install mkosi from your distribution with its package manager or the
+development version from git. The distribution packages are the preferred way to
+install mkosi.
 
-If you just want the main branch you can run
+The development version of mkosi might require tools from the systemd main
+branch, see the [`action.yaml`](action.yaml) for what we currently use in CI.
+
+## Alternative Installation methods
+
+If you just want to give the development version of mkosi a quick spin you can run
 ```shell
-python3 -m pip install --user git+https://github.com/systemd/mkosi.git
+pipx install git+https://github.com/systemd/mkosi.git
+```
+which will transparently install mkosi into a virtual environment and a mkosi
```

Comment:
python virtual environment (to avoid confusion)

Reply:
I don't think anybody will confuse this, but sure.